app/models/entry.rb

```diff
@@ -20,6 +21,7 @@ class Entry < ActiveRecord::Base
 
   belongs_to :project, touch: true
   belongs_to :category
+  has_one :client, through: :project
```

Comment:
A belongs to through is not possible. But maybe we should add client is on entry level

Sent from my iPhone

> On 02 Jan 2015, at 19:37, Jurre <notifications@github.com> wrote:
> 
> In app/models/entry.rb:
> 
> > @@ -20,6 +21,7 @@ class Entry < ActiveRecord::Base
> >  
> >    belongs_to :project, touch: true
> >    belongs_to :category
> > +  has_one :client, through: :project
> Shouldn't this be a belongs_to association?
> 
> —
> Reply to this email directly or view it on GitHub.
>